### DISCUSSION TITLE
Is it possible to block only external rest api calls?

### DISCUSSION THREAD

**Discussion by xitex** [nicumicle/simple-jwt-login#35] [Q&A]
is it possible to block external calls but not block local ones so that the work of plugins and themes using the endpoint is not broken

**Comment by true2thepen** [comment]
The short answer is no. This is due to WordPress using javascript to call rest endpoints. This means that there isn't really a way to tell if the call is actually external to WordPress or WordPress calling the endpoint from an user's browser. The solution would be to find a way to get WordPress to add the jwt to all its rest endpoint calls. 

See this [discussion](https://github.com/nicumicle/simple-jwt-login/discussions/72).
